var/spack/repos/builtin/packages/py-jcb/package.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyJcb(PythonPackage):
+    """
+    JEDI Configuration Builder
+    """
+
+    homepage = "https://github.com/NOAA-EMC/jcb"
+    git = "https://github.com/NOAA-EMC/jcb"
+
+    maintainers("danholdaway", "CoryMartin-NOAA", "AlexanderRichert-NOAA")
+
+    license("GPL-3.0-only", checked_by="AlexanderRichert-NOAA")
+
+    version("develop", branch="develop", commit="16399323e36df6f17bfd4740a2330ca7fae31537")
+
+    depends_on("python@3.6:", type=("build", "run"))
+    depends_on("py-setuptools", type="build")
+
+    depends_on("py-pyyaml@6:", type=("build", "run"))
+    depends_on("py-jinja2@3.1.2:", type=("build", "run"))
+    depends_on("py-click@8:", type=("build", "run"))
+
+    depends_on("py-pytest", type="test")
```

Comment:
:thinking: Should this be `@7:` based on:

```
python_requires = >= 3.6
setup_requires =
  setuptools
install_requires =
  PyYAML>=6.0
  Jinja2>=3.1.2
  click>=8.0
tests_require =
  pytest
  flake8
  flake8-import-order

[options.extras_require]
testing =
  pytest>=7
  flake8
  flake8-import-order
```

Reply:
Makes sense. Just changed.